popup.js

```diff
@@ -37,7 +41,8 @@ function getDefaultListId(url) {
             // Must be thrown from here for Chrome
             throw res;
         } else {
-            return res.lists.filter(list => list.default)[0].id;
+            const defaultList = res.lists.filter(list => list.default)[0];
+            return defaultList ? defaultList.id : getDefaultListId(url, res.next);
```

Comment:
If no defaultList was found it would be safer to check that res.next has a value (throw an exception if not). I know this should not happen but it's better not to trust and assume too much what the server/database gives you.